plugins/modules/ipa_hostgroup.py

```diff
@@ -20,6 +20,12 @@
   diff_mode:
     support: none
 options:
+  append:
+    description:
+    - If C(yes), add the listed I(host) to the I(hostgroup).
+    - If C(no), only the listed I(host) will be in I(hostgroup), removing any other hosts.
+    default: no
```

Comment:
Always use `true` and `false` for YAML booleans, **never** `yes`/`no`/any of the other ones. Please also fix the above ones.
```suggestion
    default: false
```